src/main/kotlin/no/skatteetaten/aurora/cantus/controller/commands.kt

```diff
@@ -57,4 +69,18 @@ class ImageRepoDtoAssembler(
             throw BadRequestException("Invalid Docker Registry URL url=$urlToValidate")
         }
     }
-}
\ No newline at end of file
+
+    private fun String.splitCheckNull(): ImageRepo {
```

Comment:
Hva med å kalle denne noe annet? For nå lager en et `ImageRepo` objekt fra en String.
Kanskje `toImageRepo`?

Reply:
Ja det er nok mye lurere!